dashboard/app/controllers/sections_controller.rb

```diff
@@ -6,6 +6,16 @@ def new
     return head :forbidden unless current_user&.admin
   end
 
+  def create
+    return head :forbidden unless current_user&.admin
+
+    section = Section.new(section_params)
+    section.user = current_user
+    section.save!
+
+    render json: {section: {id: section.id}}
```

Comment:
this isn't currently used, so if we'd rather just `HEAD 201` or something here, let me know

Reply:
I don't have a strong opinion here so this seems good.

My one concern is that this route is very similar to https://github.com/code-dot-org/code-dot-org/blob/7439ee06fc497031bb0ba2b0f4c2b6621e3f2a32/dashboard/app/controllers/api/v1/sections_controller.rb#L34. I'm torn because I know we want to add the ability to create multiple sections at one time eventually and it would probably be good to create a new route for that, but this feels like duplicate code :/

If we do keep this action, a small test would be great!

Reply:
there's a slight difference in the fact that i'm requiring `name` and `grades` params here, but I'm not sure how important that is. I forgot that when we talked about creating new routes, we were still planning on doing the multiple-sections stuff at that point. I guess it's probably less work and less duplicate code to reuse the old api for now.